go/runtime/host/sandbox/bwrap_test.go

```diff
@@ -0,0 +1,43 @@
+package sandbox
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBubbleWrapSandbox(t *testing.T) {
+	// Skip test if there is no bwrap binary available.
+	if _, err := os.Stat(bwrapPath); os.IsNotExist(err) {
```

Comment:
We might want to instead use `+build` tags and handle this check elsewhere (e.g. a check in the makefile) - and on CI always build with the tag - that way we could enforce that this test will always be run on CI and it wouldn't get skipped without us noticing (e.g. someone not installing bwrap on a CI agent in future and missing this). 

Similar for other places (e.g. a `test-sgx` build tag instead of relying on env vars set in the sgx_tests)

Reply:
I just changed it so that bwrap must always be available to run these tests.